File: apps/s-2-admin/src/api/bookingReference/BookingReferenceUpdateInput.ts
import { BookingWhereUniqueInput } from "../booking/BookingWhereUniqueInput";

export type BookingReferenceUpdateInput = {
  typeField?: string;
  uid?: string;
  meetingId?: string | null;
  meetingPassword?: string | null;
  meetingUrl?: string | null;
  booking?: BookingWhereUniqueInput | null;
  externalCalendarId?: string | null;
  deleted?: boolean | null;
};

File: apps/cactus-demo-dotnet/src/APIs/BookingReference/Dtos/BookingReferenceUpdateInput.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class BookingReferenceUpdateInput
{
    public int? Id { get; set; }

    public string? TypeField { get; set; }

    public string? Uid { get; set; }

    public string? MeetingId { get; set; }

    public string? MeetingPassword { get; set; }

    public string? MeetingUrl { get; set; }

    public int? Booking { get; set; }

    public string? ExternalCalendarId { get; set; }

    public bool? Deleted { get; set; }
}
